xapian-core/cluster/cluster.cc

```diff
@@ -1,8 +1,8 @@
 /** @file cluster.cc
- *  @brief Xapian::Cluster API class
+ *  @brief Cluster API
  */
-/* Copyright (C) 2016 Richhiey Thomas
- * Copyright (C) 2016 Richard Boulton
+/* Copyright (C) 2010 Richard Boulton
+ * Copyright (C) 2016 Richhiey Thomas
```

Comment:
I've added everything to a single commit and then added more commits to fix mistakes I made before. I'll make my commits more precise now on.